### PR TITLE
complete version compatible tests and config it as PR check workflow

### PR DESCRIPTION
**Pull request by zhoucheng361**

### CODE REVIEW

**Comment by davies** [comment]
--encrypt should be --encrypt-secret

**Comment by zhoucheng361** [comment]
> 

fixed.

File: .github/workflows/version_compatible.yml [hunk 1]
```diff
@@ -1,13 +1,22 @@
 name: "version-compatible-test"
 
 on:
-  schedule:
-    - cron:  '2 0 * * *'
+  push:
+    branches: [ main ]
```
Comment (davies) [inline .github/workflows/version_compatible.yml:5]: also release-*

Comment (zhoucheng361) [reply .github/workflows/version_compatible.yml]: > also release-*
fixed

**Pull request merged**